src/util.js

```diff
@@ -1,13 +0,0 @@
-var util = {
-	isEmail: function (s) {
-		var regexp = /^[-a-z0-9~!$%^&*_=+}{\'?]+(\.[-a-z0-9~!$%^&*_=+}{\'?]+)*@([a-z0-9_][-a-z0-9_]*(\.[-a-z0-9_]+)*\.(aero|arpa|biz|com|coop|edu|gov|info|int|mil|museum|name|net|org|pro|travel|mobi|[a-z][a-z])|([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}))(:[0-9]{1,5})?$/i;
-		return regexp.test(s);
-	},
-
-	isPhone: function (s) {
```

Comment:
Kompletně chybí validace čísla poskytnutá v původním zadání, kterou stačilo pouze importovat a použít as is.